app/controllers/application_controller.rb

```diff
@@ -11,9 +11,15 @@ class ApplicationController < ActionController::Base
     unless: -> { self.class == HighVoltage::PagesController }
   )
   before_action :sections
-  # after_action :verify_authorized, unless: -> { devise_controller? }
+  after_action(
+    :verify_authorized,
+    unless: lambda do
+      devise_controller? ||
+      self.class == HighVoltage::PagesController
+    end
+  )
   # after_action :verify_policy_scoped, unless: -> { devise_controller? }
```

Comment:
Remove this commented line for now. I'm starting to feel we won't need scopes for this release :)